docker-storage-setup.conf

```diff
@@ -1,7 +1,28 @@
+# By default no new volume and fs will be setup for docker root dir. Docker
+# will create /var/lib/docker/ and use underlying fs for storing image and
+# containers.
+# 
+# If one wants to carve out a separate logical volume for storing docker
+# metadata and images/containers/volumes, then set DOCKER_ROOT_VOLUME=yes
+#
+# This will try to create a logical volume, create filesystem on that volume
+# and mount on $DOCKER_ROOT_DIR.
+DOCKER_ROOT_VOLUME=yes
+
+# The desired size for the docker root volume. It defaults using 40% of all
+# free space.
+#
+# DOCKER_ROOT_VOLUME_SIZE can take values acceptable to "lvcreate -L" as well
+# as some values acceptable to to "lvcreate -l". If user intends to pass
+# values # acceptable to "lvcreate -l", then only those values which
+# contains "%" in syntax are acceptable.  If value does not contain "%" it
+# is assumed # value is suitable for "lvcreate -L".
+DOCKER_ROOT_VOLUME_SIZE=40%FREE
+
```

Comment:
These are new options. Instead of adding them in the beginning of file, how about adding them towards the end of the file?